internal/abci/cometbft/peers.go

```diff
@@ -1,10 +1,183 @@
 package cometbft
 
 import (
+	"context"
+	"fmt"
+	"sync"
+
 	"github.com/cometbft/cometbft/crypto/ed25519"
 	"github.com/cometbft/cometbft/p2p"
+	"github.com/kwilteam/kwil-db/common/sql"
+	"github.com/kwilteam/kwil-db/internal/sql/versioning"
+)
+
+const (
+	p2pSchemaName = "peers" // TODO: Umm, should this be part of the chain metadata schema?
```

Comment:
> // TODO: Umm, should this be part of the chain metadata schema?


No, chain metadata is included as part of consensus (affects app-hash), but peers are a local db that is specific to each node